python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -62,6 +62,168 @@ def log(*args):
         print(args)
 
 
+class VarStruct(object):
```

Comment:
why no used Variable?

Reply:
I just need a simple way to record the properties at Python, I do not create them, do not to examine them.

the ```Variable ``` is too complex to use.

Reply:
When I create a Variable, I need a ```block```, and it will invoke some methods in ```C++``` and do some check. I  just need a simple way to record the variable's properties at Python.